### PR TITLE
Fix rendering of local charm name and icon

### LINKED ISSUE #515: Clicking on model name results in "Something has gone wrong"
**Describe the bug**
Version Juju 2.8-rc2 (snap) / Dashboard 0.1.3 (beta)

When I click the name of my non-controller model in the Models area, in this case "gm-dev"...

Screenshot from 2020-05-27 09-16-58

... I get the following error message...

Screenshot from 2020-05-27 09-16-18

This happens every time, above screenshot includes devtools console in case that helps.

This was with an upgrade from 2.7.6 to 2.8-rc2, following upgrade guide from https://discourse.juju.is/t/juju-2-8-rc2-release-notes/3061, which had no problems. Then running `juju upgrade-gui` to pick up new dashboard.

The `gm-dev` model is a simple deploy of https://github.com/dshcherb/cockroachdb-operator on LXD with no real use as such but all seems to be working fine.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to Models area of dashboard.
2. Click on model name other than "controller"
3. Boom!

**Expected behaviour**
After clicking on the model I'd expect to see something not unlike I see for the controller model...

Screenshot from 2020-05-27 09-32-36

**Desktop (please complete the following information):**
 - OS: Elementary 5.1
 - Browser Firefox 76.0.1

Of note, whenever I refresh the dashboard I have to log in again.

### LINKED ISSUE #527: Use default icon for local charms
We're not currently able to display local charm icons, can we create and add a default icon?
Screen Shot 2020-05-28 at 7 17 32 PM

Related: #438

### LINKED ISSUE #534: Navigation dividers visible when sidebar is closed

### PR DESCRIPTION
**Pull request by hatched**
## Done

When a model contained a local charm it would break the rendering because it was receiving the name in an incorrect format. This renders a default charm icon when it's a local charm for the time being and fixes a styling issue with the closed sidebar.

## QA

- Bootstrap a controller on a public cloud using the juju rc2 snap.
- Deploy a local charm.
 - The one used in testing `https://github.com/dshcherb/cockroachdb-operator`
 - Clone that repo.
 - Then from within the cloned directory `juju deploy .`
- `juju dashboard` and view the built in dashboard to accept the self-signed cert.
- Pull code
- Open config.js and change: 
  - `baseControllerURL` to the ip address and port (if any) from above.
  - `isJuju` to `true`
  - `identityProviderAvailable` to `false`
- `./run`
- Open http://0.0.0.0:8036/
- View the default model, it should show you the model details as per the design below.

## Details

Fixes #515 
Fixes #527
Fixes #534

## Screenshots

Screen Shot 2020-05-29 at 3 56 41 PM

### CODE REVIEW

File: src/components/ModelTableList/OwnerGroup.test.js [hunk 1]
```diff
@@ -40,12 +40,13 @@ describe("OwnerGroup", () => {
       </MemoryRouter>
     );
     const tables = wrapper.find("MainTable");
-    expect(tables.length).toBe(5);
-    expect(tables.get(0).props.rows.length).toEqual(5);
-    expect(tables.get(1).props.rows.length).toEqual(6);
-    expect(tables.get(2).props.rows.length).toEqual(1);
-    expect(tables.get(3).props.rows.length).toEqual(3);
-    expect(tables.get(4).props.rows.length).toEqual(2);
```
Comment (anthonydillon) [inline src/components/ModelTableList/OwnerGroup.test.js:48]: This is not a good way of testing. We should find a way to reference these without index.

File: src/pages/Models/Details/ModelDetails.test.js [hunk 2]
```diff
@@ -219,4 +219,20 @@ describe("ModelDetail Container", () => {
...
+      <Provider store={store}>
+        <MemoryRouter initialEntries={["/models/admin/local-test"]}>
+          <TestRoute path="/models/*">
+            <ModelDetails />
+          </TestRoute>
+        </MemoryRouter>
+      </Provider>
+    );
+    expect(
+      wrapper.find(".model-details__apps tr[data-app='cockroachdb']").length
```
Comment (anthonydillon) [inline src/pages/Models/Details/ModelDetails.test.js:235]: This is much less likely to falsely fail in the future

**Comment by spencerbygraves** [comment]
@anthonydillon I can't see the demo, does it now use the icon from here? - https://github.com/canonical-web-and-design/jaas-dashboard/issues/527

**Comment by spencerbygraves** [comment]
Added +1 assuming the icon [here](https://github.com/canonical-web-and-design/jaas-dashboard/issues/527) will be added soon.

**Comment by hatched** [comment]
@spencerbygraves I've updated it to render the default icon and added a screenshot to show you what it looks like.

**Pull request merged**